Sources/UI/Experiences/Models/BlockTapBehavior.swift

```diff
@@ -18,7 +18,7 @@ public enum BlockTapBehavior {
 // MARK: AttributeRepresentable
 
 extension BlockTapBehavior: AttributeRepresentable {
```

Comment:
`AttributeRepresentable` has been removed in this PR. You'll need to update this to match the approach you took in the `Beacon` and `Geofence` classes.